src/components/shared/binaryCheckRadio.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+export const BinaryCheckRadio = (props) => {
+  const { label, option1, option2, onChange, type, status } = props;
+  return (
+    <div className="form-group">
+      <label>{label}</label>
+      <div
+        className="custom-control custom-radio"
+        onClick={e => onChange(e, true)}
```

Comment:
do you know the difference between `onClick={e => onChange(e, true)}` and `onClick={this.onChangeCheckRadio}` ?

Reply:
:( i am doing some test and i forget to remove the div click :/